packages/slate-editor/src/modules/decorations/useDecoration.ts

```diff
@@ -0,0 +1,16 @@
+import type { Decorate } from '@prezly/slate-commons';
+import { noop } from '@technically/lodash';
+import { useContext, useEffect } from 'react';
+
+import { decorationsContext } from './context';
+
+export function useDecoration(decorate: Decorate | undefined): void {
+    const { attach } = useContext(decorationsContext);
+
+    return useEffect(() => {
```

Comment:
This `return` seems redundant, since there is nothing to really return

Reply:
56852fe352041753230e071214a0a075dc6b3bd1